Dockerfile

```diff
@@ -8,72 +8,68 @@ FROM node:20-alpine as web-builder
 ARG JELLYFIN_WEB_VERSION=master
 RUN apk add curl git zlib zlib-dev autoconf g++ make libpng-dev gifsicle alpine-sdk automake libtool make gcc musl-dev nasm python3 \
  && curl -L https://github.com/jellyfin/jellyfin-web/archive/${JELLYFIN_WEB_VERSION}.tar.gz | tar zxf - \
+ && apk del curl \
  && cd jellyfin-web-* \
  && npm ci --no-audit --unsafe-perm \
  && npm run build:production \
  && mv dist /dist
 
-FROM debian:stable-slim as app
+FROM debian:bookworm-slim as app
 
 # https://askubuntu.com/questions/972516/debian-frontend-environment-variable
 ARG DEBIAN_FRONTEND="noninteractive"
 # http://stackoverflow.com/questions/48162574/ddg#49462622
 ARG APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
 # https://github.com/NVIDIA/nvidia-docker/wiki/Installation-(Native-GPU-Support)
+ENV NVIDIA_VISIBLE_DEVICES="all"
 ENV NVIDIA_DRIVER_CAPABILITIES="compute,video,utility"
 
+ENV JELLYFIN_DATA_DIR=/config
+ENV JELLYFIN_CACHE_DIR=/cache
+
 # https://github.com/intel/compute-runtime/releases
-ARG GMMLIB_VERSION=22.0.2
-ARG IGC_VERSION=1.0.10395
-ARG NEO_VERSION=22.08.22549
-ARG LEVEL_ZERO_VERSION=1.3.22549
+ARG GMMLIB_VERSION=22.3.11.ci17747749
+ARG IGC_VERSION=1.0.15136.4
+ARG NEO_VERSION=23.35.27191.9
+ARG LEVEL_ZERO_VERSION=1.3.27191.9
```

Comment:
Might need another update, there was a new release some days ago.

Reply:
Actually it's the latest update available
![image](https://github.com/jellyfin/jellyfin/assets/68083474/59c5b95d-08f0-4be7-8305-94b49de3c88d)

Reply:
My bad I thought of the wrong repo (https://github.com/intel/media-driver)